app/views/delete_entry_confirmation.scala.html

```diff
@@ -24,18 +24,22 @@
         h1: Heading1,
         details: ProtectedUserRecordDetails,
         govukButton: GovukButton,
-        govukWarningText: GovukWarningText
+        govukWarningText: GovukWarningText,
+        formTemplate: FormWithCSRF,
+        input: GovukInput
 )
 
 @(protectedUserRecord: ProtectedUserRecord)(implicit request: Request[_], messages: Messages)
+    @mainLayout(
+        pageTitle = Messages("confirm.delete.entry.title"),
+    ) {
+        @h1("confirm.delete.entry.header")
+        @details(protectedUserRecord)
+        @govukWarningText(WarningText(content = Text(Messages("confirm.delete.warning"))))
+        @formTemplate(action = controllers.routes.DeleteEntryController.delete(), args = Symbol("autocomplete") -> "off", Symbol("novalidate") -> "") {
+            @input(Input(id = "entryId", name = "entryId", inputType = "hidden", value = Some(protectedUserRecord.entryId)))
```

Comment:
Delete form doesn't need an `<input>` element. You can continue passing that through the URL as before this change.